pack.rb

```diff
@@ -1,308 +1,30 @@
 class Array
   #  call-seq:
-  #     arr.pack( aTemplateString ) -> aBinaryString
-  #     arr.pack( aTemplateString, buffer: aBufferString ) -> aBufferString
+  #    pack(template, buffer = nil) -> string
```

Comment:
`buffer` is a keyword arg, not a positional one.

```ruby
irb(main):002:0> [1, 2, 3].pack("i*", "hi")
<internal:pack>:143:in `pack': wrong number of arguments (given 2, expected 1) (ArgumentError)
	from (irb):2:in `<main>'
	from /Users/peter/.rubies/ruby-3.1.2/lib/ruby/gems/3.1.0/gems/irb-1.4.1/exe/irb:11:in `<top (required)>'
	from /Users/peter/.gem/ruby/3.1.2/bin/irb:25:in `load'
	from /Users/peter/.gem/ruby/3.1.2/bin/irb:25:in `<main>'
irb(main):003:0> [1, 2, 3].pack("i*", buffer: "hi")
=> "hi\u0001\u0000\u0000\u0000\u0002\u0000\u0000\u0000\u0003\u0000\u0000\u0000"
```

```suggestion
  #    pack(template, buffer: nil) -> string
```